spec/datadog/tracing/distributed/trace_context_spec.rb

```diff
@@ -209,15 +213,18 @@
             "\u0000", # First unicode character
             "\u001F", # Last lower invalid character
             ',',
-            ':',
             ';',
-            "\u007F", # First upper invalid character
+            '~', # First upper invalid character
```

Comment:
Minor: May be worth noting here that `~` is `\u007E` (I had to check the manpage ;)

Reply:
You didn't memorize the Ascii table? 😅 
Adding a comment here.

Reply:
I should memorize it yeah, may come in handy if I ever get stranded on Mars.